cmd/zetae2e/local/local.go

```diff
@@ -382,6 +383,24 @@ func localE2ETest(cmd *cobra.Command, _ []string) {
 	if testPerformance {
 		eg.Go(ethereumDepositPerformanceRoutine(conf, deployerRunner, verbose, e2etests.TestStressEtherDepositName))
 		eg.Go(ethereumWithdrawPerformanceRoutine(conf, deployerRunner, verbose, e2etests.TestStressEtherWithdrawName))
+		eg.Go(
+			solanaDepositPerformanceRoutine(
+				conf,
+				deployerRunner,
+				verbose,
+				e2etests.TestStressSolanaDepositName,
+				e2etests.TestStressSPLDepositName,
+			),
+		)
```

Comment:
Would be nice if we could parallelize the SOL and SPL deposits. Running them serially seems to be what's slowing down the performance tests a lot.